zk_toolbox/crates/zk_inception/src/commands/ecosystem/init.rs

```diff
@@ -119,6 +119,7 @@ pub async fn run(args: EcosystemInitArgs, shell: &Shell) -> anyhow::Result<()> {
             genesis_args: genesis_args.clone().fill_values_with_prompt(&chain_config),
             deploy_paymaster: final_ecosystem_args.deploy_paymaster,
             l1_rpc_url: final_ecosystem_args.ecosystem.l1_rpc_url.clone(),
+            port_offset: PortOffset::from_chain_id(chain_config.id as u16).as_u16(),
```

Comment:
Implement `From<PortOffset>` for `u16`.

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/261518714409056ddd1c6c9cab823cf343bf8706